src/frequenz/client/microgrid/_component_data.py

```diff
@@ -390,7 +382,7 @@ def from_proto(cls, raw: PbComponentData) -> Self:
         raw_power = raw.inverter.data.ac.power_active
         inverter_data = cls(
             component_id=raw.id,
-            timestamp=raw.ts.ToDatetime(tzinfo=timezone.utc),
```

Comment:
Hmm we have functions for the conversion in the client library

Reply:
This comes from the SDK, predates all the recent work we've been doing with APIs.